docs/standards/tokens/LSP8-Identifiable-Digital-Asset.md

```diff
@@ -109,11 +141,9 @@ If the sender and recipient are smart contracts that implement the LSP1 standard
 
 ### `force` mint and transfer
 
-:::success
-
-It is advised to set the `force` boolean to `false` when transferring or minting tokens to avoid sending them to the wrong address.
+:::success Recommendation
 
-For instance, if the wrong address was pasted by mistake by the user in the input field of a dApp.
+It is advised to set the `force` boolean to `false` by default when transferring or minting tokens to prevent sending them to the wrong address. For instance, if the wrong address was pasted by mistake by the user in the input field of a dApp.
```

Comment:
```suggestion
When transferring or minting tokens, setting the `force` boolean to `false` is advised to limit token transactions to [ERC725 Accounts](..//universal-profile/lsp0-erc725account.md). This will ensure tokens won't be lost when sent to the wrong EOA or contract address.
```

Reply:
Addressed but wrote an alternative sentence. I prefer phrasing it the other way around, as the word "limit" looses the emphasis on the purpose of the `force` parameter.